apps/i18n/common/en_us.json

```
@@ -1724,6 +1724,8 @@
   "ltiSectionSyncDialogDescription": "New to Code.org? A [section]({aboutSectionsUrl}) is used to manage your classroom and assign curricula. We've identified the following sections & students associated with your course.\n\nNot seeing all of your sections? Make sure you are enrolled as a teacher in each section you want to sync. Learn more [about syncing here]({aboutSyncingUrl}).",
   "ltiSectionSyncDialogDescriptionNoChange": "You're all set! Your sections & students are already up to date.\n\nYou can re-sync your sections at any time. Learn more [about syncing here]({aboutSyncingUrl})",
   "ltiSectionSyncDialogError": "We encountered an error when trying to sync with your LMS.",
+  "ltiSectionSyncDialogErrorCodeLabel": "Error Code",
```

While "Error Code" is a more universal value, providing the full text gives the translator additional context, which may affect the meaning in certain languages:
```suggestion
  "ltiSectionSyncDialogErrorCode": "Error Code: {code}",
```